spec/features/takeaway.rb

```diff
@@ -0,0 +1,105 @@
+# class Takeaway
+#
+#   CHECKOUT_ERROR = "Total cost entered does not match the sum of your order!"
+#   ITEM_ERROR = "This item does not exist!"
+#
+#   attr_reader :menu, :order
+#
+#   def initialize(menu = Menu.new, order = Order.new)
+#     @menu = menu
+#     @order = order
+#   end
+#
+#   def read_menu
+#     @menu.read
+#   end
+#
+#   def order(itm, qty=1)
+#     @menu.dishes[itm].nil? ? (raise ITEM_ERROR) : @order.add_to_basket(itm, qty)
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)